File: NETMouse - .NET release/Examples/C#/Matrices_4.cs
using ABCNET.Extensions;
using ABCNET.Utils;
using System;

namespace TestProgram
{
    internal static class Program
    {
        public static Tuple<int, int> Min(this int[,] matrix)
        {
            if (matrix == null)
                throw new ArgumentNullException("matrix");

            int min = int.MaxValue;
            int minRow = 0;
            int rowsCount = matrix.GetLength(0);
            int colsCount = matrix.GetLength(1);
            for (int i = 0; i < rowsCount; i++)
                for (int j = 0; j < colsCount; j++)
                    if (min > matrix[i, j])
                    {
                        min = matrix[i, j];
                        minRow = i;
                    }

            return Tup.New(min, minRow);
        }

        private static void Main(string[] args)
        {
            Matr.ReadInteger(Base.ReadInteger("N:"), Base.ReadInteger("M:"), "Элемент ({0}, {1})-ый:").Min().Println();
        }
    }
}

File: NETMouse - .NET release/Examples/PascalABC.NET/Matrices_4.pas
{$reference ABCNET.dll}
uses ABCNET.Utils;
uses ABCNET.Extensions;
uses System;

function Min(self: array [,] of integer): (integer, integer); extensionmethod;
begin
  if self = nil then
    raise new ArgumentNullException('self');

  var _min := integer.MaxValue;
  var minRow := 0;
  var rowsCount := self.GetLength(0);
  var colsCount := self.GetLength(1);
  for var i := 0 to rowsCount - 1 do
    for var j := 0 to colsCount - 1 do
      if _min > self[i, j] then
      begin
        _min := self[i, j];
        minRow := i;
      end;
  
  result := (_min, minRow);
end;

begin
  Matr.ReadInt32(Base.ReadInt32('N:'), Base.ReadInt32('M:'), 'Элемент ({0}, {1})-ый:').Min().Println();
  Base.ReadChar();
end.